src/Uno.UI.Wasm/WasmCSS/Uno.UI.css

```diff
@@ -143,3 +143,7 @@ body {
 textarea {
   resize: none;
 }
+
+::-ms-clear {
```

Comment:
maybe it should target texboxes directly?

Reply:
Is there any case where we want to show microsoft specific inputs?

Reply:
At this point, let's target textboxes for now, we'll adjust for others as needed.